muikku-core/src/main/java/fi/otavanopisto/muikku/security/MuikkuPermissions.java

```diff
@@ -79,7 +79,11 @@
   @Scope (PermissionScope.ENVIRONMENT)
   @DefaultEnvironmentPermissionRoles ({ EnvironmentRoleArchetype.ADMINISTRATOR, EnvironmentRoleArchetype.MANAGER, EnvironmentRoleArchetype.STUDY_PROGRAMME_LEADER, EnvironmentRoleArchetype.TEACHER, EnvironmentRoleArchetype.STUDY_GUIDER })
   public static final String ACCESS_EVALUATION = "ACCESS_EVALUATION";
-
+  
+  @Scope (PermissionScope.ENVIRONMENT)
+  @DefaultEnvironmentPermissionRoles ({ EnvironmentRoleArchetype.ADMINISTRATOR, EnvironmentRoleArchetype.MANAGER, EnvironmentRoleArchetype.STUDY_PROGRAMME_LEADER, EnvironmentRoleArchetype.TEACHER, EnvironmentRoleArchetype.STUDY_GUIDER })
```

Comment:
Is TEACHER necessary or would it be sufficient if they can access the activity log of the students at the courses they are teaching at (which has different permission)?

Reply:
@anttiviljakainen This needs additional grooming, especially the question of which courses in guider -> current student do load from a teacher perspective. If only those he teach, then ok, but if all of them are loaded, it is more complicated. Should muikku provide empty data for other courses, or not show chart icon at all? or there should be data on all the courses for teachers?

Reply:
@anttiviljakainen This needs additional grooming, especially the question of which courses in guider -> current student do load from a teacher perspective. If only those he teach, then ok, but if all of them are loaded, it is more complicated. Should muikku provide empty data for other courses, or not show chart icon at all? or there should be data on all the courses for teachers?

Reply:
@anttiviljakainen This needs additional grooming, especially the question of which courses in guider -> current student do load from a teacher perspective. If only those he teach, then ok, but if all of them are loaded, it is more complicated. Should muikku provide empty data for other courses, or not show chart icon at all? or there should be data on all the courses for teachers?

Reply:
This needs additional grooming, especially the question of which courses in guider -> current student do load from a teacher perspective. If only those he teach, then ok, but if all of them are loaded, it is more complicated. Should muikku provide empty data for other courses, or not show chart icon at all? or there should be data on all the courses for teachers?

Reply:
This needs additional grooming, especially the question of which courses in guider -> current student do load from a teacher perspective. If only those he teach, then ok, but if all of them are loaded, it is more complicated. Should muikku provide empty data for other courses, or not show chart icon at all? or there should be data on all the courses for teachers?

Reply:
This needs additional grooming, especially the question of which courses in guider -> current student do load from a teacher perspective. If only those he teach, then ok, but if all of them are loaded, it is more complicated. Should muikku provide empty data for other courses, or not show chart icon at all? or there should be data on all the courses for teachers?

Reply:
Decided to go with teacher environment permission for now. though the methods for workspace permissions are kept in place